polkadot/zombienet_tests/elastic_scaling/0003-slot-based-3cores.toml

```diff
@@ -0,0 +1,48 @@
+[settings]
+timeout = 1000
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.async_backing_params]
+  max_candidate_depth = 6
+  allowed_ancestry_len = 2
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.scheduler_params]
+  max_validators_per_core = 1
+  num_cores = 4
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+default_command = "polkadot"
+
+  [relaychain.default_resources]
+  limits = { memory = "4G", cpu = "3" }
+  requests = { memory = "4G", cpu = "3" }
+
+  [[relaychain.node_groups]]
+  name = "elastic-validator"
+  count = 6
+  args = [ "-lparachain=debug,parachain::candidate-backing=trace,parachain::provisioner=trace,parachain::prospective-parachains=trace,runtime=debug"]
+
+# First collator uses elastic scaling MVP.
+[[parachains]]
+id = 2100
+chain = "elastic-scaling-mvp"
+add_to_genesis = true
+
+  [[parachains.collators]]
+  name = "collator-elastic-mvp"
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["-laura=trace,runtime=info,cumulus-consensus=trace,consensus::common=trace,parachain::collation-generation=trace,parachain::collator-protocol=trace,parachain=debug", "--force-authoring", "--experimental-use-slot-based"]
+
+# Second collator uses UMP commitments (RFC 103).
```

Comment:
This test makes `0001` obsolete. Why not just replace it ?

Reply:
not really, since it uses adder-collator, not a cumulus-based one, which uses the collator_fn instead of SubmitCollation and also does not have the UMP signal